src/java/org/httpkit/HttpUtils.java

```diff
@@ -257,6 +257,14 @@ public static String getHost(URI uri) {
         }
         return host;
     }
+    
+    public static String getProxyHost(URI uri){
```

Comment:
This is also a good point.  Does clj-http infer the scheme for a request-level proxy override from the request uri scheme?

Also, should I add a feature-request for http-kit to respect the [Java "standard" networking properties](https://docs.oracle.com/javase/8/docs/api/java/net/doc-files/net-properties.html)?